openpype/pipeline/delivery.py

```diff
@@ -294,17 +296,30 @@ def hash_path_exist(myPath):
     src_head = src_collection.head
     src_tail = src_collection.tail
     uploaded = 0
+    first_frame = next(iter(src_collection.indexes))
```

Comment:
Maybe use `min()` here as well?

Reply:
Using `min` will be slower however. The `indexes` are sorted so we should always be taking the first entry instead of requiring to iterate the full sorted set.

Reply:
I suggested it so it's like in the `load/delivery` but maybe there it's required cause they might not be in order (?).